client/src/App.js

```diff
@@ -7,9 +7,9 @@ function App() {
 
   //Fetch API allows for asyncronous http requests and returns a promise.
   React.useEffect(() => {
-    fetch("/showtracker")
+    fetch("/api")
```

Comment:
We will want to update this soon, because we aren't pulling anything from the DB with the /api endpoint. that just returns our "under construction" message. It's fine for now, but we'll want to make that change soon 👍